source/Preferences.cpp

```diff
@@ -119,6 +119,9 @@ namespace {
 	const vector<string> BOARDING_SETTINGS = {"proximity", "value", "mixed"};
 	int boardingIndex = 0;
 
+	const vector<string> FLOTSAM_SETTINGS = {"off", "on", "flagship only", "escorts only"};
+	int flotsamIndex = 2;
```

Comment:
```suggestion
	int flotsamIndex = 1;
```
it should be "on" by default